code/datums/components/food/edible.dm

```diff
@@ -411,12 +411,12 @@ Behavior that's still missing from this component that original food items had t
 		return
 	var/mob/living/L = user
 	if(bitecount == 0 || prob(50))
-		L.manual_emote("nibbles away at \the [parent]")
+		L.manual_emote("nibbles away at \the [parent].")
 	bitecount++
 	. = COMPONENT_ITEM_NO_ATTACK
 	L.taste(owner.reagents) // why should carbons get all the fun?
 	if(bitecount >= 5)
-		var/sattisfaction_text = pick("burps from enjoyment", "yaps for more", "woofs twice", "looks at the area where \the [parent] was")
-		if(sattisfaction_text)
-			L.manual_emote(sattisfaction_text)
+		var/satisfaction_text = pick("burps from enjoyment.", "yaps for more!", "woofs twice.", "looks at the area where \the [parent] was.")
+		if(satisfaction_text)
+			L.manual_emote(satisfaction_text)
```

Comment:
```suggestion
		L.manual_emote(pick("burps from enjoyment.", "yaps for more!", "woofs twice.", "looks at the area where \the [parent] was."))
```

Reply:
the if is unnecessary and adds unnecessary indentation, which we try our best to avoid. You could alternatively still assign ``satisfaction_text`` to a var prior to emoting if you really wanted to, but I don't really see any other reason to do this.